src/popup/services/constants.service.ts

```diff
@@ -0,0 +1,11 @@
+import { ConstantsService } from 'jslib/services/constants.service';
+
+export class LocalConstantsService extends ConstantsService {
+    static readonly disableKonnectorsSuggestions: string = 'disableKonnectorsSuggestions';
```

Comment:
```suggestion
    static readonly disableKonnectorsSuggestionsStorageKey: string = 'disableKonnectorsSuggestions';
```

I suggest making it explicit that this and `konnectorSuggestionLastExecution` are just storage keys and not the actual value (as opposed to `konnectorSuggestionInterval `)

Reply:
I added the `Key` (without storage) to be consistent with existing consts in jslib